connectors/memory/memory.go

```diff
@@ -115,17 +115,14 @@ func findInsertionPoint(pk *dosa.PrimaryKey, data []map[string]dosa.FieldValue,
 	return
 }
 
-// compareRows compares two maps of row data based on clustering keys. It handles ascending/descending
-// based on the passed-in schema
+// compareRows compares two maps of row data based on clustering keys.
 func compareRows(pk *dosa.PrimaryKey, v1 map[string]dosa.FieldValue, v2 map[string]dosa.FieldValue) (cmp int8) {
 	keys := pk.ClusteringKeys
 	for _, key := range keys {
 		d1 := v1[key.Name]
 		d2 := v2[key.Name]
 		cmp = compareType(d1, d2)
-		if key.Descending {
-			cmp = -cmp
-		}
+
```

Comment:
I don't understand why this code was removed. I believe it's necessary for the case where you have a mix of ascending and descending columns in the clustering keys.

Reply:
+1 for clarification. 

Even with a single clusterKey, we'll need it for for correctly comparing the target row with the existing rows. E.g. if the clustering key sorts the rows in DESC, we'll need to find the first row that's smaller than or equal to the target row's clustering key.

Reply:
Updated the diff. There are some more issues in the memory conector